### DISCUSSION TITLE
How to represent CORE_CHARGE in Adempiere pricing system

### DISCUSSION THREAD

**Discussion by sdgathman** [adempiere/adempiere#4119] [Q&A]
Items like a battery have a "core charge" that is refunded to the customer when they bring back the used up item.  This applies to many other kinds of items that can be remanufactured or refurbished. 

There are 2.4 million items to import.  I am building up I_Product.  For starters, I think I need to add a PriceCore field to the I_Product table.  Then the actual import needs to add to Price List somehow.   In this case, the core charge refunded to customer is also received from the manufacturer when the product is returned.  (They might be different in general.)  Deposits on valuable recycled items (like glass) have similar accounting.

**Comment by marcalwestf** [comment]
hi @sdgathman Stuart.
I read your explanation several times.
I suppose your reasoning is correct. 

Another way to solve it is implementing it as an Asset. This way you don't change the Product, but what you are dealing with, i.e. Assets.

Perhaps it is not a functionality worth integrating into the ADempiere core because it is a very specific task.
